easybuild/easyconfigs/w/wtdbg2/wtdbg2-2.5-GCCcore-9.3.0.eb

```
@@ -0,0 +1,37 @@
+# Author:: Michael Dickens <cmdickens@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'MakeCp'
+
+name = 'wtdbg2'
+version = '2.5'
+
+homepage = 'https://github.com/ruanjue/wtdbg2'
+
+description = """
+ Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio
+ or Oxford Nanopore Technologies (ONT). It assembles raw reads without error
+ correction and then builds the consensus from intermediate assembly output.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/ruanjue/wtdbg2/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a2ffc8503d29f491a9a38ef63230d5b3c96db78377b5d25c91df511d0df06413']
+
+builddependencies = [('binutils', '2.34')]
+
+modextrapaths = {
+    'PATH': 'scripts',
+}
+
```

Currently the `Makefile` of `wtdbg` ignores compilation flags set in the environment. It should be fixed so that the compilation of `wtdbg` follows what is set by EasyBuild
```suggestion

prebuildopts = "sed -i 's/CFLAGS=-g3/CFLAGS+=-g3/g' Makefile && "

```